descriptors/ModuleDescriptor-template.json

```
@@ -169,6 +169,29 @@
       "permissionName": "fqm.query.async.results.get",
       "displayName": "FQM - Retrieve results of a submitted query",
       "description": "Retrieve results of a submitted query",
+      "visible":  true,
+      "subPermissions": [
+        "fqm.query.async.results.post",
+        "fqm.query.async.results.sortedids.get",
+        "fqm.query.async.results.query.get"
+      ]
+    },
+    {
+      "permissionName": "fqm.query.async.results.sortedids.get",
+      "displayName": "FQM - Retrieve results of a submitted query with sorted Ids ",
+      "description": "Retrieve results of a submitted query with sorted Ids",
+      "visible":  true
+    },
+    {
+      "permissionName": "fqm.query.async.results.query.get",
+      "displayName": "FQM - Retrieve results of a submitted query",
+      "description": "Retrieve results of a submitted query",
+      "visible":  true
+    },
+    {
+      "permissionName": "fqm.query.async.results.post",
+      "displayName": "FQM - Retrieve results of a submitted query",
+      "description": "Retrieve results of a submitted query",
```

This description feels wrong. Maybe something like this?
```suggestion
      "description": "Submit an a query to run asynchronously",
```